cloud-data-protection/src/components/home.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import React, { Component } from "react";
```